src/db.rs

```diff
@@ -104,13 +104,13 @@ impl DbController {
 
     pub async fn search_city(
         &self,
-        n: &String,
-        c: &String,
-        s: &String,
+        n: &str,
```

Comment:
maybe let's not use shorthands

Reply:
[no shorthands](https://github.com/pxp9/weather_bot_rust/pull/9/commits/d515ab63d21893e0c95a894f6fef4b38586c16a8)